### PR TITLE
PLENTY-219_configure_price_as_text

### PR DESCRIPTION
**Pull request by TheKeymaster**

### CODE REVIEW

File: resources/views/ItemList/Components/Filter/ItemFilterPrice.twig [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<script type="x/template" id="vue-findologic-item-price">
```
Comment (dblaichinger) [inline resources/views/ItemList/Components/Filter/ItemFilterPrice.twig]: Just for my understanding, why didn't we need this for Ceres 4.6? Did Plenty remove this template in 5.0?

Comment (dblaichinger) [reply resources/views/ItemList/Components/Filter/ItemFilterPrice.twig]: And was this copied or did you write it yourself?

File: tests/Mocks/PlentyRequestMock.php [hunk 2]
```diff
@@ -198,4 +198,11 @@ public function getQueryString(): string
     public function getMethod(): string
     {
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function getLocale(): string
+    {
+    }
```
Comment (dblaichinger) [inline tests/Mocks/PlentyRequestMock.php]: Is this related to the price filter? Or a change which sneaked in?

File: .travis.yml [hunk 3]
```diff
@@ -13,4 +13,4 @@ install:
   - php composer.phar install
 
 script:
-  - php vendor/bin/phpunit -c tests/phpunit.xml
\ No newline at end of file
+  - composer test
```
Comment (dblaichinger) [inline .travis.yml]: Keeping the travis yml clean and simple, I like it :)

File: resources/views/ItemList/Components/Filter/ItemFilterPrice.twig (hunk 1 above)
Comment (TheKeymaster) [reply resources/views/ItemList/Components/Filter/ItemFilterPrice.twig]: > Just for my understanding, why didn't we need this for Ceres 4.6? Did Plenty remove this template in 5.0?

Plenty now uses `.vue` files. We manually need to override these to be able to use them as before.

> And was this copied or did you write it yourself?

The code was copied straight from plenty. It will be removed once we also switch to `.vue` files.

File: tests/Mocks/PlentyRequestMock.php (hunk 2 above)
Comment (TheKeymaster) [reply tests/Mocks/PlentyRequestMock.php]: I have updated the dependency `plentymarkets/plugin-interface`. Since the abstract class (which this one extends from) added this method additionally as `abstract` I also needed to add it here.

File: .travis.yml (hunk 3 above)
Comment (TheKeymaster) [reply .travis.yml]: :+1:

File: resources/views/ItemList/Components/Filter/ItemFilterPrice.twig (hunk 1 above)
Comment (dblaichinger) [reply resources/views/ItemList/Components/Filter/ItemFilterPrice.twig]: Please create a story with all necessary information to convert our twig files to vue. Also mentioning the files which can be removed like this one. When this is done this discussion can be resolved
